axelrod/random_.py

```diff
@@ -12,3 +12,9 @@ def random_choice(p=0.5):
     if r < p:
         return Actions.C
     return Actions.D
+
+def randrange(a, b):
+    """Python 2 / 3 compatible randrange."""
```

Comment:
Yup, I know why it's here. Just asking for a 1 line saying: `returns a random float between a and b (inclusive)`.